docs/docs/1.1.2/policies/external-services.md

```diff
@@ -1,10 +1,19 @@
 # External Service
 
-This policy enables consuming services that are not part of the mesh from services running inside it. The default [passthrough](/docs/1.1.2/policies/mesh/#controlling-the-passthrough-mode) cluster will allow for accessing any non-mesh host by its domain name or IP address. As the name of the feature suggests, this pattern leaves the mesh administrator with no tools to aply any policies for such traffic. Therefore, ExternalService resource allows for declaring the desired external resource as a named service within the mesh and enabling the observability, security and traffic manipulation similar to any other service in the mesh.
+This policy enables consuming services that are not part of the mesh from services running inside it. The default 
+[passthrough](/docs/1.1.2/policies/mesh/#controlling-the-passthrough-mode) cluster will allow for accessing any non-mesh host by 
+its domain name or IP address. As the name of the feature suggests, this pattern leaves the mesh administrator with no tools 
+to aply any policies for such traffic. Therefore, ExternalService resource allows for declaring the desired external resource 
+as a named service within the mesh and enabling the observability, security and traffic manipulation similar to any other service in the mesh.
```

Comment:
This content's been bothering me for a while -- let's clean it up:
`This policy allows services running inside the mesh to consume services that are not part of the mesh. The `ExternalService` resource allows you to declare specific external resources by name within the mesh, instead of implementing the default [passthrough mode](/docs/1.1.2/policies/mesh/#controlling-the-passthrough-mode). Passthrough mode allows access to any non-mesh host by specifying its domain name or IP address, without the ability to apply any traffic policies. The `ExternalService` resource enables the same observability, security, and traffic manipulation for external traffic as for services entirely inside the mesh.`

Reply:
Then remove the tip at lines 9-12 and replace with a simple paragraph:

To enable this policy, [disable passthrough mode](mesh/#controlling-the-passthrough-mode) for the mesh and enable the [data plane proxy builtin DNS](../networking/dns/#data-plane-proxy-built-in-dns) name resolution.